vagrant/dev/bootstrap.sh

```diff
@@ -20,6 +20,7 @@ yum install -y \
 	docker \
 	python-devel python-tox \
 	python-virtualenv python-virtualenvwrapper python-pip \
-	libffi-devel \
+	enchant \
```

Comment:
Because this is unrelated and came up in the release review I am separating this out - see FLOC-1565